app/src/renderer/scripts/num.js

```diff
@@ -59,10 +59,27 @@ function uatoms(number = 0) {
   return BigNumber(number).times(1e6).toString()
 }
 
+// convert micro denoms like uatom to display denoms like atom
+function denom(denom) {
```

Comment:
I would change the name here mostly because it's a name used across the codebase, and the function is doing something with the denom, like normalize or demicrodenominatorizer XD

Reply:
i agree with karoly. would be better to have a more explicit name. 

not `demicrodenominatorizer` tho LOL

Reply:
done